src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapTheNewGeneration.java

```diff
@@ -30,55 +30,55 @@ private AdsBaseObject openPageWithVideoInLocalStorage(Page page) {
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsVuapTngDesktop")
-  public void vuapDefaultStateShouldHaveLinkToProject43ArticleOnImage(Page page, String slot, String videoIframeSelector) {
-    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
-    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
-    scrollToSlot(slot, ads);
-
-    vuap.clickOnClickArea2();
-
-    final String actual = ads.switchToNewBrowserTab();
-    Assert.assertTrue(actual.equals(PROJECT43_TNG_ARTICLE_LINK), "Image should point to page on project43.");
-  }
-
-  @Test(groups = {"AdsVuapDefaultStateTng", "AdsVuapDefaultStateClickOasis"},
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsVuapTngDesktop")
-  public void vuapDefaultStateShouldHaveLinkToFandomArticleOnImage(Page page, String slot, String videoIframeSelector) {
-    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
-    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
-    scrollToSlot(slot, ads);
-
-    vuap.clickOnClickArea4();
-
-    final String actual = ads.switchToNewBrowserTab();
-    Assert.assertTrue(actual.equals(FANDOM_ARTICLE_WESTWORLD_LINK), "Image should point to page on fandom.");
-  }
+//  @Test(groups = {"AdsVuapDefaultStateTng", "AdsVuapDefaultStateAutoplayOasis"},
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "adsVuapTngDesktop")
+//  public void vuapDefaultStateShouldStartPlayingAdvertisementAutomatically(Page page, String slot, String videoIframeSelector) {
+//    AdsBaseObject ads  = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    scrollToSlot(slot, ads);
+//
+//    VuapAssertions.verifyVideoPlay(vuap);
+//  }
+//
+//  @Test(groups = {"AdsVuapDefaultStateTng", "AdsVuapDefaultStateTimeProgressOasis"},
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "adsVuapTngDesktop")
+//  public void vuapDefaultStateShouldProgressInTime(Page page, String slot, String videoIframeSelector) {
+//    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    scrollToSlot(slot, ads);
+//
+//    VuapAssertions.verifyVideoTimeIsProgressing(vuap);
+//  }
+//
+//  @Test(groups = {"AdsVuapDefaultStateTng", "AdsVuapDefaultStateClickOasis"},
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "adsVuapTngDesktop")
+//  public void vuapDefaultStateShouldHaveLinkToProject43ArticleOnImage(Page page, String slot, String videoIframeSelector) {
+//    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    scrollToSlot(slot, ads);
+//
+//    vuap.clickOnClickArea2();
+//
+//    final String actual = ads.switchToNewBrowserTab();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)